rasterio/path.py

```diff
@@ -20,12 +20,13 @@
     'file': 'file',
     'oss': 'oss',
     'gs': 'gs',
+    'swift': 'swift',
```

Comment:
Unless we find some precedent on the internet for "swift" as a URI scheme, I think we should remove this for now.

Reply:
fixed